src/Compilers/VisualBasic/Portable/Generated/Syntax.xml.Internal.Generated.vb

```diff
@@ -103,7 +103,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
         Friend Sub New(ByVal kind As SyntaxKind, empty As InternalSyntax.PunctuationSyntax)
             MyBase.New(kind)
-            MyBase._slotCount = 1
```

Comment:
vb overrode this, instead of using the existing SlotCount property.  C# uses SlotCount.  This brings the generated code in sync.